bson/binary.py

```diff
@@ -66,6 +66,16 @@
     from mmap import mmap as _mmap
 
 
+_NUMPY_AVAILABLE = False
+try:
+    import numpy as np
+    import numpy.typing as npt
+
+    _NUMPY_AVAILABLE = True
+except ImportError:
+    np = None  # type: ignore
```

Comment:
IIRC importing numpy is very very slow. In that case we should not even attempt to import it by default.

Reply:
I've changed to lazy imports, and used `importlib.util.find_spec("numpy")` as skip condition in `test_bson.py`.